app/models/user.js

```diff
@@ -16,8 +16,9 @@ export default Owner.extend({
   token: attr('string'),
   secureUserHash: attr('string'),
   gravatarId: attr('string'),
-  firstLoggedInAt: attr('date'),
+  firstLoggedInAt: attr(),
   allowMigration: attr('boolean'),
+  recentlySignedUp: attr(''),
```

Comment:
Same here about `attr('')`

Reply:
@sorta Great finding. Thank you. I already corrected it.